pkgs/applications/networking/cluster/kubebuilder/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
+, makeWrapper
+, git
+, go
+}:
+
+buildGoModule rec {
+  pname = "kubebuilder";
+  version = "3.1.0";
+
+  src = fetchFromGitHub {
+    owner = "kubernetes-sigs";
+    repo = "kubebuilder";
+    url = "https://github.com/kubernetes-sigs/kubebuilder.git";
```

Comment:
```suggestion
```

This is not required and is build together from owner and repo.

Reply:
Removed